### DISCUSSION TITLE
How to import additional reducers?

### DISCUSSION THREAD

**Discussion by laterinc** [reduxjs/redux-toolkit#2534] [Q&A] [answered]
I start by creating an Redux store.
`export const store = configureStore({
  reducer: {
      global: globalReucer,
      counter: counterReducer,
  },
})`
How to import additional reducers?
`store.replaceReducer()` ?

**Comment by markerikson** [comment, marked as answer]
We've got instructions on how to do code-splitting of reducers in our docs over here:

https://redux.js.org/usage/code-splitting

That said, most of the time this isn't a thing you should need to worry about :)  Reducers are pretty small size-wise.

**Reply by laterinc** [reply]
Thanks!
